app/ui/src/test/java/com/fsck/k9/fragment/AccountRetrieverTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.fsck.k9.fragment
+
+import android.database.Cursor
+import com.fsck.k9.Account
+import com.fsck.k9.Preferences
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import org.mockito.ArgumentMatchers.anyInt
+import org.mockito.ArgumentMatchers.anyString
+import org.mockito.BDDMockito.given
+import org.mockito.Mockito.mock
+
+class AccountRetrieverTest {
+
+    @Test fun shouldRetrieveAccountCorrectly() {
+        // given
+        val someUuid = "someUuid";
+        val expectedAccount = Account(someUuid)
+        val preferences = mock(Preferences::class.java)
```

Comment:
`mockito-kotlin` makes this nicer to read (and write). See e.g. https://github.com/k9mail/k-9/blob/f73ef35892d5085f3ffde109563cdae07da7578c/app/core/src/test/java/com/fsck/k9/message/quote/TextQuoteCreatorTest.kt#L24-L33

Reply:
Outdated, tested class was removed.